app/models/stacks_file.rb

```diff
@@ -15,12 +15,6 @@ def file_exist?
     path && File.exist?(path)
   end
 
-  def valid?
-    # FIXME: image_exist? is defined in app/models/concerns/djatoka_adapter.rb,
-    #  which is included in StacksImage, but is probably undefined here
-    image_exist?
-  end
```

Comment:
everything about this method feels odd.  There is a separate file_exist?  method ... and I can imagine how rails magic might be happier with a "valid?" method that calls file_exist? ... but we don't seem to use valid?  and it certainly makes no sense as it was.